source/unified-test-format/tests/invalid/expectedTracingSpans-missingPropertySpans.yml

```
@@ -0,0 +1,13 @@
+description: "expectedTracingSpans-missingPropertySpans"
+
+schemaVersion: "1.26"
+
+createEntities:
+  - client:
+      id: &client0 "client0"
+
+tests:
+  - description: "missing required property spans"
+    operations: []
+    expectTracingMessages:
+      client: client0
```

```suggestion
      client: *client0
```

Per an earlier comment, this is where you'd use YAML references. Otherwise, there's no point in defining a `&client0` anchor above (but we should for consistency).